scripts/artifacts-building/containers/patches/repo_server

```diff
@@ -20,3 +20,6 @@ index 7fb1949..34cb748 100644
    tags:
      - pkg-repo-user
      - repo-key
+-- 
+2.11.0 (Apple Git-81)
```

Comment:
It's output from the fresh patch I put together. It replaces whatever your patch was previously.